src/components/applications/kanban/Consents.tsx

```diff
@@ -79,17 +80,44 @@ const Consents: React.FC<ConsentsProps> = ({ handleClose, dialog }) => {
   const [expand, setExpand] = useState(false)
   const [filtersOn, setFiltersOn] = useState(false)
   const [resetFilters, setResetFilters] = useState(false)
-  const { deleteConsentDialog, deleteUnid } = useSelector((state: AppState) => state.consents)
+  const { deleteConsentDialog, appName, username, scope } = useSelector((state: AppState) => state.consents)
+
+  const getUserName = (input: string): string | '' => {
+    let str = "username" as keyof typeof input;
+    const user = input[str];
+    const match = user.toString().match(/CN=(.*?)\//);
```

Comment:
Tried opening Consents (both dialog and page), it only showed a blank page and this error:

<img width="769" alt="image" src="https://github.com/HCL-TECH-SOFTWARE/domino-rest-adminclient/assets/130198272/0fcc3f02-c8f3-4872-a8f3-bf9dd7f1d6f5">

Reply:
resolve by changing  with ?.toString()